core-common/src/main/resources/event-type-schema/_metadata/_metadata.3.avsc

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "Metadata",
+  "type": "record",
+  "namespace": "org.zalando.nakadi.generated.avro",
```

Comment:
Added namespace as well, which will be taken as package when generating stubs

Reply:
where usually generated classes go? do they have a separate module?

Reply:
They are not generated and they dont have a separate module currently. The temporary solution is user of generated classes will have a copy of the .avsc file and will generate the classes locally.